src/css/profile/wearable/changeable/theme-circle/theme.colorPalette.less

```diff
@@ -0,0 +1,104 @@
+/**
+  Color palette
+**/
+@primary-color: #008cff;
+@primary-light_color: #4cacff;
+@white-color: #ffffff;
+@black-color: #000000;
+@bg-normal-color: #002a4d;
+@bg-press-color: #004680;
+@bg-dim-color: #3d3d3d;
+@controller-color: #4297ff;
+@controller-off-color: #8f8f8f;
+@edit-icon-color: #b82e2e;
+@fast-scroll-bg-color: fade(#3880d9, 83%);
+@index-scroll-bg-color: fade(#404040, 95%);
+@list-bg-select-color: #3d89cc;
+@list-check-color: @controller-color;
+@list-check-stroke-color: @white-color;
+@list-check-stroke-width: 2;
+
+@more-icon-dim-color: #cccccc;
+@number-bg-color: #002a4d;
+@number-bg-dim-color: #575757;
+@number-bg-press-color: #0f5d99;
+
+@rotary-bg-2nd-color: #808080;
+@rotary-bg-3rd-color: #666666;
+@rotary-bg-dim-color: #4d4d4d;
+@rotary-bg-press-color: #bfbfbf;
+@rotary-icon-dim-color: #333333;
+@rotary-icon-dim-02-color: #878787;
+@rotary-icon-color: #080808;
+@rotary-icon-press-color: #167ed9;
+@screen-reader-focus-stroke-color: #ff9100;
+@screen-reader-focus-stroke-width: 2;
+@scroll-color: #999999;
+@scroll-press-color: #0088ff;
+@select-all-color: #003e70;
+@select-all-press-color: #0063b3;
+@sub-title-bar-color: #cccccc;
+
```

Comment:
please move section progress up so this will be in alhabetic order

Reply:
ok